### ISSUE TITLE
Update FAQ & CG re sharing private info, questions frequently asked on freshdesk

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1187]
**TL;DR:** We wondered what info to provide specifically to sys admins/ schools re setting up Scratch in #1007. That sort of grew into what info can we provide for general Tech FAQ-like topics from freshdesk. I added a lot of canned responses to freshdesk to address those topics. W.r.t. information security/ safety, we might want to clarify our existing FAQ & CG (as well as having the longer, more detailed responses to specific questions I added to freshdesk). @kaschm already thumbs-up'd these changes, but I thought other people - @SpeakkVisually, @thisandagain, potentially others as well - should approve before any changes are made to the FAQ/ CG.

These suggestions result from https://github.com/LLK/scratch-www/issues/1007.

### Updating FAQ/ CG page

We already cover the information at a high level that I was considering adding to the FAQ. I don't know if it makes sense to include the details in the FAQ, but maybe we can re-word some of the existing FAQ questions to be a little stronger/ more specific.

**Existing FAQ & CG**
https://scratch.mit.edu/info/faq/
https://scratch.mit.edu/community_guidelines/

> **What information can I share on / with my account?**
Please don’t share personal contact information, such as your physical address, email, phone number, or anything else that can be used to make contact outside of the Scratch website. Please report projects, comments, or forum posts that contain this kind of information so the Scratch team can remove it, and remind the author of our policy.

> **Is it OK to have more than one person logged into an account?**
This is discouraged, because the website and project editor can easily get confused when more than one person is logged into the same account.

> **What information do you ask for during account registration?**
To protect the privacy of our community members, we limit what we collect and what we publish on the website. During the registration process, we ask for the following information:
> * username - We ask that users avoid using their real names or other identifying information.
country
> * birth month and year - We use this to confirm ownership of the account if the owner loses the password and email or asks to close an account.
> * gender
> * contact email address - If the account holder is younger than 13, we ask for the email address of their parent or guardian. We do not send email to this address except when someone requests to have the account password reset.

> The username and country of the account holder are displayed publicly on their profile page. The birth month / year, email address, and gender associated with the account are not displayed publicly. We collect this info so we can know the age and gender of our users in aggregate, and for research purposes. We do not sell or rent information about our users to anyone.

(From the CG)
> **Keep personal info private.**
For safety reasons, don't give out any information that could be used for private communication - such as real last names, phone numbers, addresses, email addresses, links to social media sites or websites with unmoderated chat.

**Proposed FAQ & CG Changes**

> **What information can I share on / with my account?**
Please don’t share personally identifiable information, such as your name, address, email, phone number, school, links to social media sites, websites with unmoderated chat, or anything else that could be used to identify, find, or contact you outside of the Scratch website. Please report projects, comments, or forum posts that contain or request this kind of information so the Scratch team can remove it, and remind the author of our Community Guidelines (link to Community Guidelines).

> **Is it OK to share a single Scratch account with multiple people?**
This is discouraged, because the website and project editor can easily get confused when more than one person is using the same account. Because Scratch wasn’t designed with this use in mind, you are increasing your likelihood of your work not being saved.

> **What information do you ask for during account registration?**
To protect the privacy of our community members, we limit what we collect and what we publish on the website. During the registration process, we ask for the following information:
> * username
> * country
> * birth month and year - We use this to confirm ownership of the account if the owner loses the password and email or asks to delete an account.
> * gender
> * contact email address - If the account holder is younger than 13, we ask for the email address of their parent or guardian. We do not send email to this address except when someone requests to have the account password reset, or if they have opted into receiving updates from the Scratch Team.

> We collect this info so we can know the age and gender of our users in aggregate, and for research purposes. We do not sell or rent information about our users to anyone.
The username and country of the account holder are displayed publicly on the user's profile page. The birth month / year, email address, and gender associated with the account are not displayed publicly. We keep this information private as a means of keeping Scratch safe. For the same reason, we ask users to keep this information private as part of our Community Guidelines (link).

(On the CG)
> **Keep personal info private.**
For safety reasons, don't give out any information that could be used to identify, find, or contact you outside of Scratch. This personally identifiable information includes real names, the name of your school, phone numbers, addresses, email addresses, links to social media sites or websites with unmoderated chat.
